script/cadence/scripts/get-balances.cdc

```diff
@@ -1,11 +1,13 @@
-import FlowColdStorageProxy from 0xProxy
+// Commenting out this dependency since no accounts on Mainnet contain this contract. If ColdStorage functionality is restored
+// then it's abridge to cross at that time
+//import FlowColdStorageProxy from 0xProxy
 import FlowToken from 0x0ae53cb6e3f42a79
 import FungibleToken from 0xee82856bf20e2aa6
```

Comment:
Do we want these hardcoded here? These look like emulator addresses so perhaps used for testing

Reply:
Fair point. So bizarrely these files aren't actually executed by the running system. It uses the copies of these same files in script.go instead. In future we want to refactor so that script.go is eliminated and the configuration of imports is done by pulling the cdc file content and swapping imports out based on env config